pkgs/applications/misc/fnott/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchgit
+, pkg-config
+, meson
+, ninja
+, scdoc
+, wayland-protocols
+, tllist
+, fontconfig
+, freetype
+, pixman
+, libpng
+, wayland
+, wlroots
+, dbus
+, fcft
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fnott";
+  version = "1.1.0";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/fnott.git";
+    rev = version;
+    sha256 = "1xaz13iry3fdjhqk9xzg29kbv7hqj8dzzqxr5mhrj8f2m013jdc3";
+  };
+
+  nativeBuildInputs = [ pkg-config meson ninja scdoc wayland-protocols tllist ];
```

Comment:
do you mind breaking these out to 1-per-line? a little crowded

Reply:
I have broken the inputs into separate lines, thanks